VERSIONS.md

```diff
@@ -24,7 +24,7 @@
 | [jq]                      | *internal*       | v1.6    | ![][jq-ver]                      | ![][jq-act]                      |
 | [libxml2]                 | grafana          | v2.11.0 | ![][libxml2-ver]                 |                                  |
 | [memcached_exporter]      |                  | v0.15.2 | ![][memcached_exporter-ver]      | ![][memcached_exporter-act]      |
-| [mongodb_exporter]        |                  | v1.0.0  | ![][mongodb_exporter-ver]        | ![][mongodb_exporter-act]        |
+| [mongodb_exporter]        |                  | v0.44.0  | ![][mongodb_exporter-ver]        | ![][mongodb_exporter-act]        |
```

Comment:
Can you please check line 143. I think there we need an adjustment as well to show the correct latest versions afterwards.